### DISCUSSION TITLE
[Question] How To send a Push Notification In Flet

### DISCUSSION THREAD

**Discussion by Siva-Rajan-R** [flet-dev/flet#4237] [Q&A]
### Question

I want to use FCM to send a Push Notification. I don't know how to implement it in Flet . Is there any one has an idea.please share it 🙏

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Ayberk-Irmak** [comment]
Hey everyone! 👋

I finally found a solution for sending notifications in Flet mobile applications using Pyjnius!
Since I know that many of you have been struggling with this issue, I decided to make a video tutorial explaining the entire process.
Check it out, and let me know what you think!

🔗 Video Link: [How to Send Notifications in Flet using Pyjnius](https://www.youtube.com/watch?v=kXWKaylIhCg)

Hope it helps! 😊

**Reply by Youssef3082004** [reply]
Thanks for your tutorial, it's very helpful
